### PR TITLE
Task 94 - Created Services screen + little fixes in styles

### PR DESCRIPTION
**Pull request by Kuba-Nalepa**

### CODE REVIEW

File: presentation/src/main/res/layout/fragment_services.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:text="Services"
+        android:layout_marginTop="@dimen/size_3x"/>
+
+    <com.google.android.material.tabs.TabLayout
+        android:id="@+id/tabLayout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="20dp"
```
Comment (Zalezny) [inline presentation/src/main/res/layout/fragment_services.xml:20]: dimens can be here

File: presentation/src/main/java/com/safetyheads/akademiaandroida/presentation/ui/fragments/services/ServicesFragment.kt [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            override fun onTabSelected(tab: TabLayout.Tab) {
+                viewPager.currentItem = tab.position
+            }
+
+            override fun onTabUnselected(tab: TabLayout.Tab?) {
+                //
+            }
+
+            override fun onTabReselected(tab: TabLayout.Tab?) {
+                //
```
Comment (Zalezny) [inline presentation/src/main/java/com/safetyheads/akademiaandroida/presentation/ui/fragments/services/ServicesFragment.kt:52]: I think better is //no-op

**Review by Zalezny — DISMISSED** [review]
seems ok

File: presentation/src/main/res/layout/fragment_services.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    tools:context=".ui.fragments.services.ServicesFragment">
+
+    <com.safetyheads.akademiaandroida.presentation.ui.components.H2View
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:text="Services"
```
Comment (forceindia712) [inline presentation/src/main/res/layout/fragment_services.xml:13]: I recommend putting "Services" into strings.xml

**Review by forceindia712 — CHANGES_REQUESTED** [review]

**Review by forceindia712 — APPROVED** [review]

**Review by Zalezny — APPROVED** [review]

**Pull request merged**